### DISCUSSION TITLE
Welcome to the REST APIs with Flask and Python Discussions!

### DISCUSSION THREAD

**Discussion by jslvtr** [schoolofcode-me/rest-api-sections#31] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our student community, as well as to make public what we're working on and why. We hope that you:

* Ask questions about future work or development.
* Share ideas.
* Show projects you've been working on.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Also remember, GitHub Discussions is not the place for questions about your code or your projects. Please use the Udemy Q&A or our Discord server for that (or StackOverflow!).

**Comment by Regson** [comment]
Hi, I am Regha Abraham... A mechanical engineer but have so much passion for coding.
I have worked on APIs and that's why this forum resonates with me. 
Currently building a blog site using Flask— a python framework and of course HTML and CSS. 
Can't wait to integrate and collaborate with like minds here. 
Thank you.

**Reply by jslvtr** [reply]
Hi Regha! Great to see you here, thank you! I'm looking forward to collaborating!

**Comment by resatcaner** [comment]
Hi, this is Caner from Istanbul. I am a BA/BI specialist and Innovation Team senior member in Stats Perform. I am following your courses and events Jose, to improve my coding skills. Using python for daily solutions for data analysis/ML cases. Moreover, going to develop a flask-based API for the integration of a couple of systems and communicate with field members of the company. I have to admit that I took a couple of courses, not just about REST-APIs, and have to state yours are, by far, the most beneficial and best-structured ones among others (including university curriculums.)

Looking forward to meeting with other members. Hoping to contribute as much as the business as usual allows.

Cheers

**Reply by jslvtr** [reply]
Hi Caner, thanks for commenting and for your kind words about the courses! I feel honored 🙏. I'm keen to make them even better with your help and suggestions!
